Neos.Eel/Classes/Helper/MathHelper.php

```diff
@@ -422,14 +422,14 @@ public function round($subject, $precision = 0)
      */
     public function sign($x)
     {
-        if ($x < 0) {
+        if (!is_numeric($x)) {
```

Comment:
In PHP8 'foo' compares differently to a number: see https://3v4l.org/Zhdsh
It is more sane to first check the non-numeric case and return NAN anyway, as it is done similarly in other math helper functions.